mesa/examples/advanced/emperor_dilemma/app.py

```diff
@@ -0,0 +1,115 @@
+from model import EmperorModel
+
+from mesa.experimental.devs.simulator import ABMSimulator
```

Comment:
You import the ABM simulator here, not the devssimulator as claimed in the PR. Moreover, I don't see any place where you are scheduling events, so this model can be run without the use of a simulator.